lib/rules/computed-property-getters.js

```diff
@@ -94,9 +95,25 @@ module.exports = {
       }
     };
 
+    let importedEmberName;
+    let importedComputedName;
```

Comment:
FWIW, there is nothing wrong with this (from a technical level):

```js
import Ember from 'ember';
import Em from 'ember';
import LOL from 'ember;
```

All in the same module, and IMHO ^ **should** still trigger the rule. If I understand this code correctly, we are assuming there is only one.

Reply:
While dealing with this would be a large hassle to just tack on to the current implementation, I think it would be pretty easy to handle with the `gatherImports`-style refactor.

Reply:
Will consider this in the future, although it does seem like it could be pretty inconvenient to handle this edge case. Hopefully, most people are using a rule like [import/no-duplicates](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md) to avoid this.

Reply:
Maybe, not sure. I would expect that the goal of the a higher level `gatherImports` style API is to abstract this issue away.